packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -95,226 +96,232 @@ const getTitle = (market_type: string, show_eu_related_content: boolean) => {
-                            currency={mt5_trade_account.currency}
-                            has_sign={!!mt5_trade_account.balance && mt5_trade_account.balance < 0}
-                            show_currency
-                        />
-                    </Text>
-                )}
-            </div>
-            <div className='cfd-trade-modal__login-specs'>
-                <div className='cfd-trade-modal__login-specs-item'>
-                    <Text className='cfd-trade-modal--paragraph'>{localize('Broker')}</Text>
-                    <SpecBox is_bold is_broker value={'Deriv Holdings (Guernsey) Limited'} />
-                </div>
-                <div className='cfd-trade-modal__login-specs-item'>
-                    <Text className='cfd-trade-modal--paragraph'>{localize('Server')}</Text>
-                    <SpecBox is_bold value={(mt5_trade_account as DetailsOfEachMT5Loginid)?.server_info?.environment} />
-                </div>
-                <div className='cfd-trade-modal__login-specs-item'>
-                    <Text className='cfd-trade-modal--paragraph'>{localize('Login ID')}</Text>
-                    <SpecBox is_bold value={(mt5_trade_account as TTradingPlatformAccounts)?.display_login} />
-                </div>
-                <div className='cfd-trade-modal__login-specs-item'>
-                    <Text className='cfd-trade-modal--paragraph'>{localize('Password')}</Text>
-                    <div className='cfd-trade-modal--paragraph'>
-                        <PasswordBox
-                            platform='mt5'
-                            onClick={() => {
-                                const account_type = getCFDAccountKey({
-                                    market_type: mt5_trade_account.market_type,
-                                    sub_account_type: mt5_trade_account.sub_account_type,
-                                    platform: CFD_PLATFORMS.DMT5,
-                                    shortcode: mt5_trade_account.landing_company_short,
-                                });
-                                onPasswordManager(
-                                    mt5_trade_account?.login,
-                                    getTitle(mt5_trade_account.market_type || '', show_eu_related_content),
-                                    mt5_trade_account.account_type || '',
-                                    account_type,
-                                    (mt5_trade_account as DetailsOfEachMT5Loginid)?.server
-                                );
-                                toggleModal();
-                            }}
+                <div className='cfd-trade-modal__login-specs'>
+                    <div className='cfd-trade-modal__login-specs-item'>
+                        <Text className='cfd-trade-modal--paragraph'>{localize('Broker')}</Text>
+                        <SpecBox is_bold is_broker value={'Deriv Holdings (Guernsey) Limited'} />
```

Comment:
Since you're doing refactoring, can you help to reuse specbox from Components folder instead of redefining the same component here 🙏🏼

Reply:
Its not in the scope of the card but i will do it.